docs/sources/developers/plugins/create-a-grafana-plugin/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+description: How-to topics for plugin development
+title: Create a Grafana plugin
+---
+
+This section contains how-to topics for developing Grafana plugins.
```

Comment:
It would be good for these to have a summary description similar to how you did in the main index, could be a follow up

Reply:
Yes, these are just stubs that I will expand upon later.